src/firmware_new/tools/BUILD

```diff
@@ -8,3 +8,10 @@ sh_binary(
     restricted_to = ["//cc_toolchain:stm32h7"],
     deps = ["@bazel_tools//tools/bash/runfiles"],
 )
+
+sh_binary(
+    name = "regen",
```

Comment:
nit: Can we name this something slightly more specific, like `regen_cubemx`? (Same goes for the `regen.sh` file)

Running a target called `//firmware_new/tools:regen` doesn't really let me know what it's doing.

Reply:
yeah agreed, changed to `cubemx_regen`